lib/ex_format.ex

```diff
@@ -1,1371 +1,25 @@
 import Kernel, except: [to_string: 1]
+alias ExFormat.{
+  Formatter,
+  State,
+  AST,
+  Comments,
+  Lines,
+}
```

Comment:
`import`s, `alias`es and friends all usually go inside the module, so let's do the same here.